tests/apache/livy/operators/test_livy.py

```diff
@@ -0,0 +1,134 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Same here, this needs to go

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/64a37920cde8a8ad44463a7d48115a088edc20d8)